File: ex10/ex10_01/src/coffee.py
class Coffee:
    def __init__(self, name, roast, price):
        self.name = name
        self.roast = roast
        self.price = price
    def get_name(self):
        return self.name
    def get_roast(self):
        return self.roast
    def get_price(self):
        return self.price
    def set_name(self, name):
        self.name = name
    def set_roast(self, roast):
        self.roast = roast
    def set_price(self, price):
        self.price = price

File: ex10/ex10_01/src/coffee.cpp
#include "coffee.hpp"

Coffee::Coffee(const string& name, const string& roast, float price)
    : name(name), roast(roast), price(price) {}

string Coffee::get_name() const {
    return name;
}

void Coffee::set_name(string new_name) {
    name = new_name;
}

string Coffee::get_roast() const {
    return roast;
}

void Coffee::set_roast(string new_roast) {
    roast = new_roast;
}

float Coffee::get_price() const {
    return price;
}
void Coffee::set_price(float new_price) {
    price = new_price;
}